### ISSUE TITLE
[Bug]: Philips Hue Outdoor Motion Sensor -> Set Sensitivity not working

### ISSUE THREAD

**Issue by nukleuz80** [Jey-Cee/ioBroker.deconz#280]
### What happened?

Set sensitivity is not working.

### Adapter Version

1.3.19

### deConz (Phoscon) Version

2.13-2 beta

### JS-Controller Version

3.3.18

### Node JS Version

12.22.1

### NPM Version

6.14.21

### Operating System

Debian

### Relevant log output

```shell
deconz.0
2021-12-13 20:07:45.104	warn	{"address":"/sensors/97/config/sensitivity","description":"invalid value, 1, for parameter sensitivity","type":7}

deconz.0
2021-12-13 20:07:03.896	warn	{"address":"/sensors/97/config/sensitivity","description":"invalid value, 0, for parameter sensitivity","type":7}

deconz.0
2021-12-13 19:53:00.201	warn	{"address":"/sensors/97/config/sensitivity","description":"invalid value, 0, for parameter sensitivity","type":7}
```

### Anything else?

If i tried to set the sensitivity i got warning in the logs. But when i try it directly with the API (Postman) it works fine and the value was set correctly.

**Comment by holomekc** [comment]
Same for temperature offset:

2022-01-07 09:18:22.304	warn	{"address":"/sensors/6/config/offset","description":"invalid value, 0, for parameter offset","type":7}

**Comment by Jey-Cee** [comment]
I fixed this, please install from Github. I need Feedback.

**Comment by Jey-Cee** [comment]
By the way it is the same as #269 & #268

**Comment by holomekc** [comment]
For sensitivity and temperature I cannot see an error anymore after updating. Also the deconz api shows that values are adjusted properly. So looks good to me. Thank you very much.

**Comment by Jey-Cee** [comment]
Thank you for the feedback, so now i can update the version and release it.

**Comment by Jey-Cee** [comment]
Fix in version 1.3.20
